### PR TITLE
Limit phone number to ten digits (DEV-435)

### PR DESCRIPTION
**Pull request by davelindqvist**
[DEV-435](https://betterangels.atlassian.net/browse/DEV-435?atlOrigin=eyJpIjoiNGEyOGYzOWU3ODc4NDczYmFmZTc4NDBhMDg0YzdhNzEiLCJwIjoiaiJ9)

phone-number-input

[DEV-435]: https://betterangels.atlassian.net/browse/DEV-435?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by davelindqvist** [comment]
[Per Yuko's response, the product team would like to add a validation message for the phone number.](https://betterangels-group.slack.com/archives/C06FEV8SARY/p1722281635033569)

Therefore, this may need to wait for approval?

**Review by mikefeldberg — APPROVED** [review]

File: libs/expo/betterangels/src/lib/screens/AddEditClient/ContactInfo.tsx
Comment (mikefeldberg) [inline libs/expo/betterangels/src/lib/screens/AddEditClient/ContactInfo.tsx:74]:
```suggestion
          minLength={10}
```

Comment (mikefeldberg) [inline libs/expo/betterangels/src/lib/screens/AddEditClient/ContactInfo.tsx:74]:
```suggestion
          minLength={10}
          maxLength={10}
```

**Pull request closed without merging**
